src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/InvokeWebRequestCommand.CoreClr.cs

```diff
@@ -38,6 +38,8 @@ internal override void ProcessResponse(HttpResponseMessage response)
             ArgumentNullException.ThrowIfNull(response);
             TimeSpan perReadTimeout = ConvertTimeoutSecondsToTimeSpan(OperationTimeoutSeconds);
             Stream responseStream = StreamHelper.GetResponseStream(response, _cancelToken.Token);
+            string outFilePath = WebResponseHelper.GetOutFilePath(response, _qualifiedOutFile);
```

Comment:
`_qualifiedOutFile` is null when `ShouldSaveToOutFile == false`, so this line of code needs to be in the `if (ShouldSaveToOutFile ) { ... }` block, cannot be put here.

Reply:
GetOutFilePath() returns null if _qualifiedOutFile is null. It doesn't any extra work. I see no point in complicating the code with an additional conditional block.

Reply:
Good point. Resolved.